cpp/daal/src/algorithms/service_kernel_math.h

```diff
@@ -378,6 +403,12 @@ class ChebyshevDistances : public PairwiseDistances<FPType, cpu>
         return services::Status();
     }
 
+    virtual services::Status getRootBatch(const size_t n, FPType * in, FPType * out)
+    {
+        out = in;
```

Comment:
It does nothing outside the function

Reply:
It's even a bug if ```in``` and ```out``` are different arrays.

Reply:
set of arguments was changed as far as it's not necessary to have in and out arrays separate.